pysat/_instrument.py

```diff
@@ -195,11 +195,10 @@ def __init__(self, platform=None, name=None, tag=None, sat_id=None,
                 self.name = inst_module.name.lower()
                 self.platform = inst_module.platform.lower()
             except AttributeError:
-                raise AttributeError(string.join(('A name and platform ',
-                                                  'attribute for the ',
-                                                  'instrument is required if ',
-                                                  'supplying routine module ',
-                                                  'directly.')))
+                raise AttributeError(' '.join(('A name and platform attribute',
```

Comment:
Shouldn't you also output the original attribute error?

```
except AttributeError as err:
    raise AttributeError(' '.join((err, '\nblah, blah')))
```

Reply:
fixed